### PR TITLE
US113777 styling the confirm page

### PR DESCRIPTION
**Pull request by TomMichielsD2L**
image
image

### CODE REVIEW

File: src/components/widget/d2l-teacher-course-creation-confirm.js [hunk 1]
```diff
@@ -1,40 +1,119 @@
...
+				border: 2px solid var(--d2l-color-sylvite);
+				background-color: var(--d2l-color-regolith);
+			}
+			.tcc-confirm__back-button {
+				margin: 0px 12px;
+			}
+			.tcc-confirm__course-type-div{
+				margin-top: 12px;
+			}
+			.tcc-confirm_back-button {
```
Comment (josephsaffron) [inline src/components/widget/d2l-teacher-course-creation-confirm.js:45]: duplicate?

Comment (TomMichielsD2L) [reply src/components/widget/d2l-teacher-course-creation-confirm.js:45]: It wasnt working. So I copied the class name from the element and did not notice I had it above.

File: src/components/widget/d2l-teacher-course-creation-welcome.js [hunk 2]
```diff
@@ -41,7 +41,7 @@ class TeacherCourseCreationWelcome extends BaseMixin(LitElement) {
 	}
 
 	_changePage() {
-		this.changePage(PAGES.INPUT_PAGE);
+		this.changePage(PAGES.CONFIRM_PAGE);
```
Comment (TomMichielsD2L) [inline src/components/widget/d2l-teacher-course-creation-welcome.js:44]: This was not supposed to be committed.

File: src/components/d2l-teacher-course-creation.js [hunk 3]
```diff
@@ -22,6 +22,7 @@ class TeacherCourseCreation extends BaseMixin(LitElement) {
 		return css`
 			:host {
 				display: inline-block;
+				width: 100%;
```
Comment (louiscoste) [inline src/components/d2l-teacher-course-creation.js:25]: Does this need to be set at 100%?

File: src/components/widget/d2l-teacher-course-creation-confirm.js [hunk 4]
```diff
@@ -1,40 +1,117 @@
+import '@brightspace-ui/core/components/button/button.js';
+import '@brightspace-ui/core/components/colors/colors.js';
 import { css, html, LitElement } from 'lit-element/lit-element';
 import { BaseMixin } from '../../mixins/base-mixin';
+import { heading2Styles } from '@brightspace-ui/core/components/typography/styles.js';
+import {inputLabelStyles} from '@brightspace-ui/core/components/inputs/input-label-styles.js';
```
Comment (louiscoste) [inline src/components/widget/d2l-teacher-course-creation-confirm.js:6]: nit: space between {} and inputLabelStyles

File: src/components/widget/d2l-teacher-course-creation-confirm.js [hunk 5]
```diff
@@ -1,40 +1,117 @@
...
+			}
 		};
 	}
 
 	static get styles() {
-		return css`
+		return [heading2Styles, inputLabelStyles, css`
 			:host {
 				display: inline-block;
+				width: 100%;
```
Comment (louiscoste) [inline src/components/widget/d2l-teacher-course-creation-confirm.js:27]: same as above, do we really need the 100%?

File: src/components/widget/d2l-teacher-course-creation-confirm.js [hunk 6]
```diff
@@ -1,40 +1,117 @@
...
 			}
 			:host([hidden]) {
 				display: none;
 			}
-		`;
+			.tcc-confirm__info-div {
+				padding: 12px 12px;
+				margin-bottom: 36px;
```
Comment (louiscoste) [inline src/components/widget/d2l-teacher-course-creation-confirm.js:34]: This is probably a good enough margin but we should revisit all the margins to make them all the same across all the screens

File: src/components/d2l-teacher-course-creation.js (hunk 3 above)
Comment (TomMichielsD2L) [reply src/components/d2l-teacher-course-creation.js:25]: yes. Flexbox sucks

Comment (louiscoste) [reply src/components/d2l-teacher-course-creation.js:25]: FLEXBOX!!!

Comment (TomMichielsD2L) [reply src/components/d2l-teacher-course-creation.js:25]: I don't want to use flexbox just to get 100% width :)

File: test/d2l-custom-teacher-course-creation-confirm.test.js [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+		});
+
+		afterEach(() => {
+			getTccServiceStub.restore();
+			fixtureCleanup();
+		});
+
+		it('finish button triggers change-page event', async() => {
+			const patches = {};
+			patches['createCourse'] = async() => 6609;
```
Comment (josephsaffron) [inline test/d2l-custom-teacher-course-creation-confirm.test.js:50]: Could inline this

File: test/d2l-custom-teacher-course-creation-confirm.test.js [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+				expect(event.detail.pageData.courseType).to.equal(pageData.courseType);
+				expect(event.detail.pageData.courseOrgUnitId).to.equal(6609);
+			});
+
+			el.shadowRoot.querySelector('.tcc-confirm__finish-button').click();
+		});
+
+		it('finish button with error triggers change-page event', async() => {
+			const patches = {};
+			patches['createCourse'] = async() => {throw new Error('Houston we have a problem');};
```
Comment (josephsaffron) [inline test/d2l-custom-teacher-course-creation-confirm.test.js:73]: Make it a const?

Comment (TomMichielsD2L) [reply test/d2l-custom-teacher-course-creation-confirm.test.js]: Done!

File: test/d2l-custom-teacher-course-creation-confirm.test.js (hunk 7 above)
Comment (TomMichielsD2L) [reply test/d2l-custom-teacher-course-creation-confirm.test.js]: I don't know how to do this.

**Pull request merged**